### PR TITLE
Add toggle for providing written statement

### PR DESCRIPTION
**Pull request by David-Feetenby**
Adds a toggle in the support console for controlling whether a teaching authority provides writtent statement or not

### CODE REVIEW

**Review by thomasleese — COMMENTED** [review]
Looks good, just left one small comment. Also I think it's worth including a screenshot in the PR description.

File: app/views/support_interface/regions/edit.html.erb [hunk 1]
```diff
@@ -9,6 +9,8 @@
 
   <%= f.govuk_check_box :application_form_enabled, 1, 0, multiple: false, link_errors: true, small: true, label: { text: "Application form enabled" } %>
 
+  <%= f.govuk_check_box :teaching_authority_provides_written_statement, 1, 0, multiple: false, link_errors: true, small: true, label: { text: "Teaching authority will only send written statement directly to TRA" } %>
```
[the remark this answers is not in the document]
Comment (David-Feetenby) [reply app/views/support_interface/regions/edit.html.erb]: Done 👍

File: app/views/support_interface/regions/edit.html.erb [hunk 2]
```diff
@@ -9,6 +9,7 @@
 
   <%= f.govuk_check_box :application_form_enabled, 1, 0, multiple: false, link_errors: true, small: true, label: { text: "Application form enabled" } %>
 
+
```
Comment (thomasleese) [inline app/views/support_interface/regions/edit.html.erb:12]: There's an extra space added here :)

```suggestion
```

File: app/views/support_interface/regions/edit.html.erb [hunk 3]
```diff
@@ -22,6 +23,8 @@
   ], :id, :name, label: { text: "Status check" } %>
 
   <%= render "shared/teaching_authority_form_fields", f: %>
+  <%= f.govuk_check_box :teaching_authority_provides_written_statement, 1, 0, multiple: false, link_errors: true, small: true, label: { text: "Teaching authority will only send written statement directly to TRA" } %>
+
```
Comment (thomasleese) [inline app/views/support_interface/regions/edit.html.erb:27]: I think the blank line should go above.

```suggestion

  <%= f.govuk_check_box :teaching_authority_provides_written_statement, 1, 0, multiple: false, link_errors: true, small: true, label: { text: "Teaching authority will only send written statement directly to TRA" } %>
```

**Review by thomasleese — APPROVED** [review]

**Pull request merged**